cli.js

```diff
@@ -24,41 +25,47 @@ const cli = meow(`
 	Run without arguments to use the interactive interface.
 `, {
 	alias: {
-		f: 'force'
+		f: 'force',
+		v: 'verbose'
 	}
 });
 
-function init() {
+function init(flags) {
 	escExit();
 
-	return psList({all: false}).then(listProcesses);
+	return psList({all: false}).then(procs => listProcesses(procs, flags));
 }
 
-function listProcesses(processes) {
+function listProcesses(processes, flags) {
 	inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
+	const verbose = flags.verbose || false;
 
 	return inquirer.prompt([{
 		name: 'processes',
 		message: 'Running processes:',
 		type: 'autocomplete',
-		source: (answers, input) => Promise.resolve().then(() => filterProcesses(input, processes))
+		source: (answers, input) => Promise.resolve().then(() => filterProcesses(input, processes, verbose))
```

Comment:
I would continue passing the `flags` object down here to make it easier to add more options in the future.